app/scripts/migrations/092.ts

```diff
@@ -0,0 +1,49 @@
+import { cloneDeep } from 'lodash';
+import { hasProperty, isObject } from '@metamask/utils';
+
+export const version = 92;
+
+/**
+ * Add ticker to the providerConfig object if missing
+ *
+ * @param originalVersionedData - Versioned MetaMask extension state, exactly what we persist to dist.
+ * @param originalVersionedData.meta - State metadata.
+ * @param originalVersionedData.meta.version - The current state version.
+ * @param originalVersionedData.data - The persisted MetaMask state, keyed by controller.
+ * @returns Updated versioned MetaMask extension state.
+ */
+export async function migrate(originalVersionedData: {
```

Comment:
this migration doesnt seem related to details/status

Reply:
There are two migrations. 93 deals with the details/status updates: https://github.com/MetaMask/metamask-extension/pull/20219/files#diff-8ebcbc75b5bcd6c7fe12e04d01226a9c4159edef463e193914a3edd2bb48ee05R8

Could package them as one but thought it made sense to separate them since the required ticker is wholly separate change.

But happy to combine if that's preferred.

Reply:
Ok I pushed an intermediate PR to update NetworkController version to 11.0.0 first: https://github.com/MetaMask/metamask-extension/pull/20367